debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
Thanks for the info @asb. I was able to successfully use git-dch to update the changelog in `debian/changelog` (with and without the dch snapshot flag), but when the deb package gets built, the changelog packed into `\usr\share\doc\mu\changelog.Debian.gz` still contains the original content without the extra version added. So, at packing it prints:

```
dpkg-buildpackage: source package mu
dpkg-buildpackage: source version 0.1-1
...
```

And when installing over a previous version the console also shows the same version:

```
Unpacking mu (0.1-1) over (0.1-1) ...
Setting up mu (0.1-1) ...
...
```

And of course `dpkg-query -l 'mu'` also shows 0.1-1.

These are the commands I am currently using, am I missing something?:
```
git-dch --ignore-branch --since="$PREV_COMMIT"
git-buildpackage --git-upstream-tree="$GIT_COMMIT" --git-ignore-branch --git-ignore-new --git-export-dir="$DEB_BUILD_DIR" --git-force-create -uc -us
```